storage/local/interface.go

```diff
@@ -26,6 +26,8 @@ import (
 // are goroutine-safe. Storage implements storage.SampleAppender.
 type Storage interface {
 	prometheus.Collector
+	Querier
```

Comment:
Right, that's an antipattern anyways. I changed it and removed the embedded interface \o/